From 0ac8385e374e238a0d50f19b8c75735e2089e6b1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Ji=C5=99=C3=AD=20Klime=C5=A1?= <jklimes@redhat.com>
Date: Wed, 18 Mar 2015 13:03:19 +0100
Subject: [PATCH] editor: remove deadcode reported by covscan
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Defect type: DEADCODE
NetworkManager-0.8.1/network-manager-applet-0.8.1/src/connection-editor/ip4-routes-dialog.c:199: dead_error_line: Execution cannot reach this statement: "goto done;".

Signed-off-by: Jiří Klimeš <jklimes@redhat.com>
---
 .../src/connection-editor/ip4-routes-dialog.c                        | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/network-manager-applet-0.8.1/src/connection-editor/ip4-routes-dialog.c b/network-manager-applet-0.8.1/src/connection-editor/ip4-routes-dialog.c
index 3e62891..aedf48c 100644
--- a/network-manager-applet-0.8.1/src/connection-editor/ip4-routes-dialog.c
+++ b/network-manager-applet-0.8.1/src/connection-editor/ip4-routes-dialog.c
@@ -191,9 +191,8 @@ validate (GtkWidget *dialog)
 			goto done;
 
 		/* Prefix */
-		if (   !get_one_prefix (model, &tree_iter, COL_PREFIX, TRUE, &prefix, NULL)
- 		    || prefix == 0)
-			goto done;
+		if (!get_one_prefix (model, &tree_iter, COL_PREFIX, TRUE, &prefix, NULL))
+                        goto done;
 		/* Don't allow zero prefix for now - that's not supported in libnm-util */
 		if (prefix == 0)
 			goto done;
-- 
2.1.0

